sources/docker-compose.yml

```diff
@@ -94,6 +102,7 @@ services:
       - REDIS_STANDALONE_MODE=${REDIS_STANDALONE_MODE}
     volumes:
       - ./packages/backend/apps/workers/src:/app/src
+      - ./packages/backend/coverage/workers:/app/apps/workers/coverage
```

Comment:
Why are we mounting the coverage path here?

Reply:
As we talked in the meeting (adding here just for history) the coverage report for E2E tests is generated inside docker and this allow us to have them available on the host. While on CI/CD it allow GitHub actions to have access to the reports and generate reports and check the tests coverage.